gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixJobLauncher.java

```diff
@@ -514,6 +599,7 @@ private TaskConfig getTaskConfig(WorkUnit workUnit, ParallelRunner stateSerDeRun
     rawConfigMap.put(GobblinClusterConfigurationKeys.TASK_SUCCESS_OPTIONAL_KEY, "true");
     TaskConfig taskConfig = TaskConfig.Builder.from(rawConfigMap);
     workUnitToHelixConfig.put(workUnit.getId(), taskConfig);
```

Comment:
same here, do we still need this? I feel you want to use helix TaskId instead of work unit Id here

Reply:
Basically seems helixIdTaskConfigMap and workUnitToHelixConfig are similar, can we just use one of them to reduce complexity?

Reply:
removed workUnitToHelixConfig